students/M_Sunday/lesson04/kata_14.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+import os
+import random
+
+
+def openfile(file):
+    text_ = []
+    f = open(file)
```

Comment:
`file` is a built-in, use alternate var names
also use `with` when working with files